msrestazure/azure_active_directory.py

```diff
@@ -757,25 +757,34 @@ def _retrieve_token_from_imds_with_retry(self):
         if self.identity_id:
             payload[self.identity_type] = self.identity_id
 
-        retry, max_retry = 1, 20
+        retry, max_retry, start_time, result = 1, 20, time.time(), None
```

Comment:
Remove `result`, no pre-declaration needed in Python. This won't pass type validation anyway ([and I'm trying to add that](https://github.com/Azure/msrest-for-python/pull/98)).